drivers/clock_control/clock_stm32_ll_h7.c

```diff
@@ -48,41 +49,6 @@ static uint32_t get_bus_clock(uint32_t clock, uint32_t prescaler)
 	return clock / prescaler;
 }
 
-static uint32_t get_flash_latency(uint32_t axi_clock)
-{
-	/* At boot flash latency is set to 7 for safety reasons */
-	/* But it can be configured once AXI clock is set */
-	/* See reference manual for wait states RM0433 embedded flash memory */
-	/* Latency is defined from the AXI RCC_ACLK clock frequency */
-	/* This is worst case frequencies for VOS3 lowest voltage scaling */
-	/* 0 to 45MHz -> LATENCY_0 */
-	/* 45 to 90MHz -> LATENCY_1 */
-	/* 90 to 135MHz -> LATENCY_2 */
-	/* 135 to 180MHz -> LATENCY_3 */
-	/* 180 to 225MHz -> LATENCY_4 */
-
-	switch (axi_clock/45000001UL) {
-	case 0:
-		return LL_FLASH_LATENCY_0;
-	case 1:
-		return LL_FLASH_LATENCY_1;
-	case 2:
-		return LL_FLASH_LATENCY_2;
-	case 3:
-		return LL_FLASH_LATENCY_3;
-	case 4:
-		return LL_FLASH_LATENCY_4;
-	case 5:
-		return LL_FLASH_LATENCY_5;
-	case 6:
-		return LL_FLASH_LATENCY_6;
-	case 7:
-		return LL_FLASH_LATENCY_7;
-	default:
-		return LL_FLASH_LATENCY_7;
-	}
-}
-
```

Comment:
Please fix this directly in the initial commit where this is introduced.
Good opportunity to check the following if you're not familiar yet:
https://blog.sebastian-daschner.com/entries/git-commit-fixup-autosquash

Reply:
Done, I squashed the 2 commits into the original one so this code won't appear as deleted in the second commit.

Reply:
This also works, txs